pkg/virt-operator/strategy_job.go

```diff
@@ -98,7 +98,7 @@ func (c *KubeVirtController) generateInstallStrategyJob(infraPlacement *v1.Compo
 		},
 	}
 
-	apply.InjectPlacementMetadata(infraPlacement, &job.Spec.Template.Spec)
+	apply.InjectPlacementMetadata(infraPlacement, &job.Spec.Template.Spec, apply.AnyNode)
```

Comment:
isn't this a mistake? Shouldn't you pass `RequireControlPlanePreferNonWorker` here?

Reply:
Yeah, I think you're right. Good catch! done